test/md055-missing-pipes.md

```diff
@@ -0,0 +1,19 @@
+# MD055 Missing Borders
```

Comment:
While there are exceptions, I prefer test files not include rule names. Something like `table-missing-pipes.md` would be good here. Also, the Heading text should match the file name (here "pipes" vs. "borders").